src/helpers/GetVisibleTodos.ts

```diff
@@ -0,0 +1,13 @@
+import { Filter } from '../types/Filter';
+import { Todo } from '../types/Todo';
+
+export function getVisibleTodos(todosList: Todo[], filterType: Filter): Todo[] {
+  switch (filterType) {
+    case Filter.Active:
+      return todosList.filter(todo => !todo.completed);
+    case Filter.Comleted:
+      return todosList.filter(todo => todo.completed);
+    default:
+      return todosList;
+  }
+}
```

Comment:
Add [empty line in the end of file](https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file)